diff --git a/public/5.7.1/wp-includes/cron.php b/public/5.7.1/wp-includes/cron.php
index 1666a09b27f..2fc7924884c 100644
--- a/public/5.7.1/wp-includes/cron.php
+++ b/public/5.7.1/wp-includes/cron.php
@@ -193,7 +193,8 @@ function wp_schedule_single_event( $timestamp, $hook, $args = array(), $wp_error
 	);
 	uksort( $crons, 'strnatcasecmp' );
 
-	return _set_cron_array( $crons, $wp_error );
+	// VIP: don't return
+	_set_cron_array( $crons, $wp_error );
 }
 
 /**
@@ -309,7 +310,8 @@ function wp_schedule_event( $timestamp, $recurrence, $hook, $args = array(), $wp
 	);
 	uksort( $crons, 'strnatcasecmp' );
 
-	return _set_cron_array( $crons, $wp_error );
+	// VIP: don't return
+	_set_cron_array( $crons, $wp_error );
 }
 
 /**
@@ -516,7 +518,8 @@ function wp_unschedule_event( $timestamp, $hook, $args = array(), $wp_error = fa
 		unset( $crons[ $timestamp ] );
 	}
 
-	return _set_cron_array( $crons, $wp_error );
+	// VIP: don't return
+	_set_cron_array( $crons, $wp_error );
 }
 
 /**
@@ -702,11 +705,15 @@ function wp_unschedule_hook( $hook, $wp_error = false ) {
 
 	$set = _set_cron_array( $crons, $wp_error );
 
+	/*
 	if ( true === $set ) {
 		return array_sum( $results );
 	}
 
 	return $set;
+	*/
+	// VIP: ignore _set_cron_array return
+	return array_sum( $results );
 }
 
 /**
